classes/class-wc-connect-api-client.php

```diff
@@ -436,30 +445,42 @@ protected function request_body( $initial_body = array() ) {
 			$default_body = array(
 				'settings' => array(),
 			);
-			$body = array_merge( $default_body, $initial_body );
+			$body         = array_merge( $default_body, $initial_body );
+
+			$helper_auth_data = WC_Connect_Functions::get_wc_helper_auth_info();
+			$access_token = '';
+			$site_id      = '';
+
+			if ( ! is_wp_error( $helper_auth_data ) ) {
```

Comment:
Because the get_wc_helper_auth_info() method can return a WP_Error I decided to set the token and site id to default empty string values and only grab their values from `$helper_auth_data` if the `get_wc_helper_auth_info()` method _doesn't_ return a WP_Error. If there's a cleaner way to do this I'm all ears. :)

Reply:
This makes sense to me!